packages/oas-to-har/__tests__/index.test.js

```diff
@@ -7,16 +7,40 @@ const commonParameters = require('./__fixtures__/common-parameters');
 
 const oas = new Oas();
 
-test('should output a har format', () => {
-  expect(oasToHar(oas)).toStrictEqual({
+expect.extend({
+  toBeAValidHAR(har) {
```

Comment:
🤯

Reply:
I'm tempted to rip this out into its own package.